azdev/utilities/command.py

```diff
@@ -33,7 +33,7 @@ def call(command, **kwargs):
     """
     return subprocess.call(
         command,
-        shell=True,
+        shell=False,
```

Comment:
`shell` by default is `False`: https://docs.python.org/3/library/subprocess.html#subprocess.run

There is no need to explicitly specify it.

Reply:
After changing `shell` to `False`, this function will break:

https://docs.python.org/3/library/subprocess.html#subprocess.Popen

> On POSIX, if args is a string, the string is interpreted as the name or path of the program to execute. However, this can only be done if not passing arguments to the program.

Reply:
Added for linux-like platform, skipped for windows.